Packs/Zendesk/Integrations/Zendeskv2/Zendeskv2.yml

```
@@ -709,6 +715,174 @@ script:
     - contextPath: Zendesk.Organization.url
       type: string
       description: The API URL of this organization.
+  - name: zendesk-group-user-list
+    description: |-
+      Get group's users. 
+      Allowed for: Admins, Agents and Light Agents.
+    arguments:
+    - name: group_id
+      required: true
+      description: |-
+        The ID of a specific group.
+    - name: limit
+      default: true
+      description: Maximum number of results to return. Default is 50.
```

This will be auto added to the description once you set a default in the yml
```suggestion
      description: Maximum number of results to return.
```